tfe/resource_tfe_organization_membership.go

```diff
@@ -0,0 +1,93 @@
+package tfe
+
+import (
+	"fmt"
+	"log"
+
+	tfe "github.com/hashicorp/go-tfe"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceTFEOrganizationMembership() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceTFEOrganizationMembershipCreate,
+		Read:   resourceTFEOrganizationMembershipRead,
+		Delete: resourceTFEOrganizationMembershipDelete,
+
+		Schema: map[string]*schema.Schema{
+			"email": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"organization": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+		},
+	}
+}
+
+func resourceTFEOrganizationMembershipCreate(d *schema.ResourceData, meta interface{}) error {
+	tfeClient := meta.(*tfe.Client)
+
+	// Get the email and organization.
+	email := d.Get("email").(string)
+	organization := d.Get("organization").(string)
+
+	// Create a new options struct.
+	options := tfe.OrganizationMembershipCreateOptions{
+		Email: tfe.String(email),
+	}
+
+	log.Printf("[DEBUG] Create membership %s for organization: %s", email, organization)
+	membership, err := tfeClient.OrganizationMemberships.Create(ctx, organization, options)
+	if err != nil {
+		return fmt.Errorf(
+			"Error creating membership %s for organization %s: %v", email, organization, err)
+	}
+
+	d.SetId(membership.ID)
+
+	return resourceTFEOrganizationMembershipRead(d, meta)
+}
+
+func resourceTFEOrganizationMembershipRead(d *schema.ResourceData, meta interface{}) error {
+	tfeClient := meta.(*tfe.Client)
+
+	log.Printf("[DEBUG] Read configuration of membership: %s", d.Id())
+	// membership, err := tfeClient.OrganizationMemberships.ReadWithOptions(ctx, d.Id(), options)
```

Comment:
nit: should this be removed? or a comment added about why we are keeping it?

Reply:
Thanks for the catch- my fault! Will remove.